web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-examples.adoc

```diff
@@ -0,0 +1,137 @@
+. Set the *Display Name* field to `Set Variable`.
+. Set *Name* to `variableSample` and *Value* to `test`.
+
+In Studio, the *Set Variable* configuration looks like this:
+
+.Set Variable Configuration
+image::web-service-consumer-example3.png[Set Variable Configuration]
+
+[start=8]
+. Drag a *Transform Message* component onto the canvas, to the right of the *Set Variable* component. +
+The *Transform Message* component transforms the variable value into XML.
+. In the *Output* field of the *Transform Message* component configuration, set the DataWeave expression that:
+* Transforms the variable value output to `application/xml`.
+* Defines the name of the web service operation that the Web Service Consumer aims to invoke `attachOperation`.
+
+.DataWeave Expression
+[source,xml,linenums]
+----
+%dw 2.0
+output application/xml
+ns con http://service.soap.service.mule.org/
+---
+con#attachOperation: {
+   text: vars.variableSample
+}
+----
+
+In Studio, the *Transform Message* configuration looks like this:
+
+.Transform Message Configuration
+image::web-service-consumer-example4.png[Transform Message Configuration]
+
+[start=10]
+. Drag the Web Service Consumer *Consume* operation onto the canvas, to the right of the *Transform Message* component. +
+. Set the *Display Name* field to `Consume`.
+. Set the *Operation* to `attachOperation` and the *Body* message to `payload`.
+. In the *Message Customization* section of the *Advanced* tab, enable the option *Force XML Prolog into body* to append the XML Prolog into the envelope's body XML content.
+
+In Studio, the *Consume* operation configuration looks like this:
+
+.Consume Operation Configuration
+image::web-service-consumer-example5.png[Consume Operation Configuration]
+
+In Studio, the *XML Prolog* configuration looks like this:
+
+.Configure XML Prolog into Body
+image::web-service-consumer-xmlprolog.png[Configure XML Prolog into Body]
+
+[start=14]
+. On the *General* tab,  click the plus sign (+) next to the *Connector configuration* field to access the *Global Element Properties* configuration.
+. On the *Connection* section of the *General* tab, configure the following parameters:
+* *WSDL location* +
+The WSDL file URL remote or local.
+* *Service* +
+The service name.
+* *Port* +
+The port name.
+* *Address* +
+The address of the web server to dispatch requests but if the *Service* and *Port* parameters don't auto-populate the web server address, you must manually provide the correct *Address*.
+. Click *OK*.
+
+In Studio, the *Global Element Properties* configuration looks like this:
+
+.Web Service Consumer Configuration
+image::web-service-consumer-example6.png[Web Service Consumer Configuration]
+
+[start=17]
+. Save your changes in Studio.
+. Deploy the application.
+
+== Web Service Consumer Example in XML
+In the *Configuration XML* tab in Studio, the XML looks as follows. If needed, change the values to reflect your environment:
```

Comment:
```suggestion
In the *Configuration XML* tab in Studio, the XML looks like this (if needed, change the values to reflect your environment):
```

Reply:
...or however Diane words it. I think it's a little different from what you had originally.